docs/getting-started/official-plugins.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Official Plugins
+sidebar_label: Official Plugins List
+slug: /plugins-list
+pagination_next: "plugins/overview"
+pagination_prev: getting-started/plugins
+---
+
+# Official Amplication Plugins
```

Comment:
@dericksozo I dont think we should call these plugins official. on the pricing pages we refer to them as community plugins

Reply:
@GreenMachine01 I just renamed this to "Community Plugins".